Installation fails with FreeTDS 1.0 on OSX

Error message:
```
clang -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/include -I/usr/local/var/pyenv/versions/2.7.11/include/python2.7 -c _mssql.c -o build/temp.macosx-10.11-x86_64-2.7/_mssql.o -DMSDBLIB
_mssql.c:18783:15: error: use of undeclared identifier 'DBVERSION_80'
    __pyx_r = DBVERSION_80;
              ^
1 error generated.
error: command 'clang' failed with exit status 1
```

[Full log of failed install command](https://github.com/pymssql/pymssql/files/287439/pymssql.txt)

Reverting to FreeTDS 0.91 via `brew unlink freetds; brew install homebrew/versions/freetds091` fixes the issue as a work-around.

--- a/src/_mssql.pyx
+++ b/src/_mssql.pyx
@@ -1708,12 +1708,10 @@ cdef int _tds_ver_str_to_constant(verstr) except -1:
         return DBVERSION_42
     if verstr == u'7.0':
         return DBVERSION_70
-    if verstr == u'7.1':
+    if verstr in (u'7.1', u'8.0'):
         return DBVERSION_71
     if verstr == u'7.2':
         return DBVERSION_72
-    if verstr == u'8.0':
-        return DBVERSION_80
     raise MSSQLException('unrecognized tds version: %s' % verstr)
 
 #######################

--- a/src/sqlfront.pxd
+++ b/src/sqlfront.pxd
@@ -106,7 +106,6 @@ cdef extern from "sqlfront.h":
     int DBVERSION_70
     int DBVERSION_71
     int DBVERSION_72
-    int DBVERSION_80
 
     ## Type Constants ##
     cdef enum:

